code/modules/mob/living/simple_animal/hostile/terror_spiders/terror_spiders.dm

```diff
@@ -339,6 +340,13 @@ GLOBAL_LIST_EMPTY(ts_spiderling_list)
 		GLOB.ts_death_last = world.time
 		GLOB.ts_count_alive_station--
 
+/mob/living/simple_animal/hostile/poison/terror_spider/proc/give_intro_text()
+	to_chat(src, "<center><span class='userdanger'>You are a Terror Spider!</span></center>")
+	to_chat(src, "<center>Work with other terror spiders in your hive to eliminate the crew and claim the station as your nest!")
```

Comment:
Needs spans

Reply:
Even with no desired class?

Reply:
Ah yeah, we default to the `notice` class in that case.

Also just noticed that this line needs a closing `</center>`.